storm-server/src/main/java/org/apache/storm/utils/ServerUtils.java

```diff
@@ -635,16 +635,16 @@ public static void unZip(File inFile, File toDir) throws IOException {
      *
      * @throws IOException
      */
+    //Used only by logviewer, considering moving to web-app?
```

Comment:
even if used only in one place, if it is a general purpose function, I think it makes sense to have in a global place.  I'd remove this comment.